jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -1,9 +1,11 @@
+import numpy as np
 import astropy.units as u
+from astropy.constants import c
```

Comment:
`c` is a little to easily to be accidently overwritten by another dev down the road. This might be more explicit.

```suggestion
from astropy import constants as const
```

Also just FYI for now, especially for testing purpose, ironically constants might change over time. So when they do (again), this is good to keep in mind: https://docs.astropy.org/en/stable/constants/index.html#collections-of-constants-and-prior-versions